### PR TITLE
New OT form for the Ad Selection API

### PR DESCRIPTION
**Pull request by captainbrosset**
**Sign up for the Ad Selection API**:

* Before/Live: https://learn.microsoft.com/en-us/microsoft-edge/web-platform/ad-selection-api
* Preview: https://review.learn.microsoft.com/en-us/microsoft-edge/web-platform/ad-selection-api?branch=pr-en-us-3298

AB#54852442

### CODE REVIEW

File: microsoft-edge/web-platform/ad-selection-api.md [hunk 1]
```diff
@@ -57,15 +57,15 @@ The **Ad Selection API** flag enables the Ad Selection API and associated featur
...
-For each top-level domain that you intend to use with the Ad Selection API, fill in the **Origin Trial Registration**  form for that domain, and then submit the form.
+For each top-level domain that you intend to use with the Ad Selection API, fill in the **Origin Trial Registration** form for that domain, and then submit the form.
 
 To sign up for the **Ad Selection API** origin trial for a domain:
 
-1. Open [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/) in a new window or tab:
+1. Open [Ad Selection API Origin Trial Registration in Microsoft Edge](https://forms.office.com/r/eBhJt58Lks) in a new window or tab.
 
-   ![The "Ad Selection API" origin trial listing at "Microsoft Edge Origin Trials"](./ad-selection-api-images/origin-trials.png)
+1. Click **Start now**.
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/ad-selection-api.md:66]:
```suggestion
1. Click the **Start now** button.
```
opportunity to paint clear mental picture of UI / controls

File: microsoft-edge/web-platform/ad-selection-api.md [hunk 2]
```diff
@@ -57,15 +57,15 @@ The **Ad Selection API** flag enables the Ad Selection API and associated featur
...
 
-1. In the **Ad Selection API** section, click the **Register** button.  The **Origin Trial Registration** form opens:
+1. Read and agree to the terms of use, and then click **Next**.  The **Ad Selection API Origin Trial Registration** form opens:
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/ad-selection-api.md:68]:
```suggestion
1. Read and agree to the terms of use, and then click the **Next** button.  The **Ad Selection API Origin Trial Registration** form opens:
```

File: microsoft-edge/web-platform/ad-selection-api.md [hunk 3]
```diff
@@ -77,8 +77,6 @@ To sign up for the **Ad Selection API** origin trial for a domain:
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/ad-selection-api.md:77]: could add detail at line 76
`* In the **Subdomain Support** section, select the **Yes** or **No** option button.`

it's possible to add a lot of detail here, per notes that were at bottom of old portal, rewritten at: step 5 & 6 in section:
https://learn.microsoft.com/en-us/microsoft-edge/origin-trials/#registering-for-an-origin-trial

Comment (mikehoffms) [reply microsoft-edge/web-platform/ad-selection-api.md:77]: here's the markdown to possibly incorp, from
GitHub\edge-developer\microsoft-edge\origin-trials\index.md --
```
1. In the **Domain** text box, enter a domain URL (such as `https://example.com`) or a subdomain URL (such as `https://beta.example.com`).

   You can configure the origin trial token to support any of the following:
   * A single domain, such as `https://example.com`, without supporting its subdomains.
   * A single subdomain, such as `https://beta.example.com`.
   * A domain, such as `https://example.com`, and its subdomains, such as `https://beta.example.com`.

   Trailing paths and query parameters aren't supported.  If a URI is entered that has a trailing path or a query parameter, such as `https://example.com/path/new-feature`, the root domain (such as `https://example.com`) or subdomain will be registered, ignoring any trailing path or query parameters.

1. Set the **Enable token for subdomains?** toggle.

   * To enable using the origin trial at a domain (such as `https://example.com`) without supporting its subdomains (such as `https://beta.example.com`), make sure the **Enable token for subdomains?** toggle is turned off.

   * To enable using the origin trial only at a specific subdomain (such as `https://beta.example.com`), make sure the **Enable token for subdomains?** toggle is turned off.

   * To enable using the origin trial at a domain (such as `https://example.com`) and its subdomains (such as `https://beta.example.com`), turn on the **Enable token for subdomains?** toggle.
```

File: microsoft-edge/web-platform/ad-selection-api.md [hunk 4]
```diff
@@ -279,13 +277,9 @@ To learn more about which API methods are available and to view example code, se
...
-1. Go to the row for the **Ad Selection API** origin trial.
-
-1. Click the **issues** button in the row.  The `privacy-preserving-ads` repo opens.
+1. Go to the [MSEdgeExplainers](https://github.com/MicrosoftEdge/MSEdgeExplainers) repo at GitHub.com.
 
-1. Submit a new issue, or click the **Issues** tab to view the issues.
+1. Submit a new issue, or click the **Issues** tab to view the existing issues.
 
 
 <!-- ====================================================================== -->
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/ad-selection-api.md:285]: probably delete line 308-310 in See Also section:
```
Origin trials:
* [Microsoft Edge Origin Trials](https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/)
```
https://microsoftedge.github.io/MSEdgeExplainers/origin-trials/

Comment (mikehoffms) [inline microsoft-edge/web-platform/ad-selection-api.md:285]: See Also section, could possibly add link to main OT article eg for maintainers - maybe add:
```
<!-- note for maintainers
* [Use origin trials in Microsoft Edge](../origin-trials/index.md)
--> 
```
https://learn.microsoft.com/microsoft-edge/origin-trials/

**Review by mikehoffms — COMMENTED** [review]
* remove old See Also link
* define "Subdomain Support"
* add hidden See Also link for main OT article, to assist maintainers
* specify that these two clicked things are buttons

**Review by mikehoffms — APPROVED** [review]
LGTM

**Review by leahmsft — APPROVED** [review]
LGTM

**Pull request merged**